resources/js/features/games/components/AchievementSetCredits/MobileCreditDialogTrigger/MobileCreditDialogTrigger.tsx

```diff
@@ -0,0 +1,279 @@
+  aggregateCredits,
+  artCreditUsers,
+  codingCreditUsers,
+  designCreditUsers,
+}) => {
+  const { t } = useTranslation();
+
+  const nonAuthorUniqueContributors = useMemo(() => {
+    return [...artCreditUsers, ...codingCreditUsers, ...designCreditUsers].filter(
+      (user, index, self) => index === self.findIndex((u) => u.displayName === user.displayName),
+    );
+  }, [artCreditUsers, codingCreditUsers, designCreditUsers]);
+
+  // Dedupe logic credits with authors - it's a bit redundant.
+  // TODO do this on the server to reduce initial props size
+  const filteredLogicCredits = aggregateCredits.achievementsLogic.filter(
+    (logicUser) =>
+      !aggregateCredits.achievementsAuthors.some(
+        (author) => author.displayName === logicUser.displayName,
+      ),
+  );
+
+  // If there's no claims or credit to show, then bail.
+  if (
+    !achievementSetClaims.length &&
+    !aggregateCredits.achievementsAuthors.length &&
+    !nonAuthorUniqueContributors.length
+  ) {
+    return null;
+  }
+
+  return (
+    <div className="flex w-full flex-col gap-x-1 sm:hidden">
+      <BaseDialog>
+        <BaseDialogTrigger asChild>
+          <BaseButton size="sm" className="w-full gap-1.5 py-[15px]">
+            {achievementSetClaims.length ? (
+              <span>
+                <LuWrench className="size-4" />
+                <span className="sr-only">{t('Claimed')}</span>
+              </span>
+            ) : null}
+
+            {!!achievementSetClaims.length && !!aggregateCredits.achievementsAuthors.length ? (
+              <span>{'·'}</span>
+            ) : null}
+
+            {aggregateCredits.achievementsAuthors.length ? (
+              <>
+                {aggregateCredits.achievementsAuthors.length < 5 ? (
+                  <UserAvatarStack
+                    canLinkToUsers={false}
+                    isOverlappingAvatars={false}
+                    maxVisible={5}
+                    size={20}
+                    users={aggregateCredits.achievementsAuthors}
+                  />
+                ) : null}
+
+                <span>
+                  {t('{{val, number}} authors', {
+                    count: aggregateCredits.achievementsAuthors.length,
+                    val: aggregateCredits.achievementsAuthors.length,
+                  })}
+                </span>
+              </>
+            ) : null}
+
+            {!!aggregateCredits.achievementsAuthors.length &&
+            !!nonAuthorUniqueContributors.length ? (
+              <span>{'·'}</span>
+            ) : null}
+
+            {nonAuthorUniqueContributors.length ? (
+              <span className="flex items-center">
+                <span>
+                  {t('+{{val, number}} contributors', {
+                    count: nonAuthorUniqueContributors.length,
+                    val: nonAuthorUniqueContributors.length,
+                  })}
+                </span>
+              </span>
+            ) : null}
+          </BaseButton>
```

Comment:
For a set that's in development, this widget shows an unlabelled wrench:
<img width="368" height="118" alt="image" src="https://github.com/user-attachments/assets/7e2b1a50-2d43-4374-9465-eea357ef42eb" />

<img width="362" height="110" alt="image" src="https://github.com/user-attachments/assets/4f66fcc1-9ebe-4d45-bbce-65dd7fd048f2" />

Reply:
Fixed in latest. To keep things simple, now there's just a "Claimed" label and a spacer dot added.

<img width="360" height="116" alt="Screenshot 2025-07-27 at 5 08 16 PM" src="https://github.com/user-attachments/assets/8e5090ee-05a3-46a5-89f9-e5fa83f8c771" />